LogMonitor/src/LogMonitor/FileMonitor/Utilities.cpp

```diff
@@ -35,9 +35,8 @@ HANDLE CreateFileMonitorEvent(
  * 
  * @return HANDLE 
  */
-HANDLE GetLogDirHandle(std::wstring logDirectory, HANDLE stopEvent) {
+HANDLE GetLogDirHandle(_In_ std::wstring logDirectory, _In_ HANDLE stopEvent, _In_ std::double_t waitInSeconds) {
```

Comment:
Seen the previous comment on the reasons for double to support `Inf`.

Reply:
We don't want to set a default value here since we have the default value being set in `LoggerSettings.h`.

Reply:
We don't want to set a default value here since we have the default value being set in `LoggerSettings.h`. We should consume whatever value is passed here in the method that calls it.